### PR TITLE
UTY-497: Pin libraries to repo

### PR DESCRIPTION
**Pull request by DanailPenev**
**Contributions**: We are not currently taking public contributions - see our [contributions](https://github.com/spatialos/UnityGDK/blob/master/CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/UnityGDK/blob/master/README.md#give-us-feedback).

-------

#### Description
Pinned libraries to repository.
#### Tests
Game works locally. Running prepare_workspace.sh still works.
#### Primary reviewers

### CODE REVIEW

**Comment by firtina-improbable** [comment]
Would be nice to have a doc on how to upgrade these if we need to

**Comment by jared-improbable** [comment]
Isn't it redundant to have `Plugins/*/Plugins/*`? I'd say drop the second `Plugins`

**Comment by firtina-improbable** [comment]
When I pulled the repo and did a codegen, there was a clash, so may need to send a message in the channel to clean the previously ignored folders.

File: workers/unity/spatialos.unity.client.build.json [hunk 1]
```diff
@@ -75,10 +75,11 @@
...
       "steps": [
         {
-          "name": "Dependencies",
+          "name": "Remove Codegen",
+          "command": "rm",
           "arguments": [
-            "worker_package",
-            "clean"
+            "-rf",
+            "./Assets/improbable/genereated/*"
```
Comment (jessicafalk) [inline workers/unity/spatialos.unity.client.build.json]: this path still has the typo in it ;)
change it to be the same as for the gamelogic build file

**Pull request merged**